one_fm/overrides/leave_application.py

```diff
@@ -668,6 +671,35 @@ def send_proposed_date_email(doc_name):
     sendemail(sender=sender, recipients= recipient,
             message=msg, subject=" Leave Application – Suggested Adjustment to Leave Dates", delayed=False, is_scheduler_email=False,is_external_mail=True)
 
+@frappe.whitelist()
+def send_leave_details_email_to_employee(self):
+    employee_info = frappe.db.get_value("Employee", self.employee, ["employee_name_in_arabic","personal_email", "company_email","prefered_email"], as_dict=1)
+    args = frappe._dict({
+                    "doc_name": self.name,
+                    "doc_type": self.doctype,
+                    "employee_name_eng" : self.employee_name,
+                    "employee_name_arabic" : employee_info.get("employee_name_in_arabic") or "",     
+                    "employee_id" : self.employee,
+                    "leave_type_eng" : self.leave_type,
+                    "from_date" : self.from_date,
+                    "to_date" : self.to_date,
+                    "total_leave_days" : self.total_leave_days,
+                    "date_of_application" : self.posting_date,
+                    "leave_approver" : self.leave_approver_name,
+                    "status":self.workflow_state,
+                    "base_url": frappe.utils.get_url()
```

Comment:
@ibtesam-onefm , you could use get url to form, so that you would pass it in directly

Reply:
Sure